### ISSUE TITLE
Hyperland-git version conflict during installation

### ISSUE THREAD

**Issue by binarykitchen** [end-4/dots-hyprland#644]
In the middle of the installation it fails due to a version mismatch between 0.5.2.r2 and 0.5.2-1

I had to resolve this packaging conflict manually by removing one of them.

You might want to look into this? Thanks

**Comment by binarykitchen** [comment]
![24-06-25 14-07-28 4239](https://github.com/end-4/dots-hyprland/assets/177641/35f11592-985b-484e-9c14-94021621cf6f)
Btw here's a snapshot ...

**Comment by unhelpfulninja** [comment]
Yeah and it's still there and it's craaaaazy how he ain't doing nun about it 😕😕

**Comment by clsty** [comment]
We're now using stable packages from Arch. Closing.
